zjit/src/hir.rs

```diff
@@ -6176,6 +6177,42 @@ mod opt_tests {
         "#]]);
     }
 
+    #[test]
+    fn test_branchnil_nil() {
+        eval("
+            def test
+                x = nil
+                x&.itself
```

Comment:
```suggestion
              x = nil
              x&.itself
```

I know it's complicated, but we don't indent Ruby code with 4 spaces while we indent the Rust code (like this string itself) with 4 spaces.